.github/workflows/element-build-and-test.yaml

```diff
@@ -88,6 +88,20 @@ jobs:
             cypress/videos
             cypress/synapselogs
 
+      - name: Store benchmark result
+        if: github.ref == 'refs/heads/develop'
+        uses: matrix-org/github-action-benchmark@jsperfentry-1
+        with:
+          name: Cypress measurements
+          tool: 'jsperformanceentry'
+          output-file-path: cypress/performance/measurements.json
+          # The dashboard is available at https://matrix-org.github.io/matrix-react-sdk/cypress/bench/
+          benchmark-data-dir-path: cypress/bench
+          fail-on-alert: false
+          comment-on-alert: false
+          github-token: ${{ secrets.DEPLOY_GH_PAGES }}
+          auto-push: ${{ github.ref == 'refs/heads/develop' }}
```

Comment:
I don't think having this superfluous condition is helpful anymore

Reply:
I agree it's now a bit redundant, but I think it's a tad safer to leave it in. It sounds like the team may soon enable some form of alerting, and it would be too easy to forget to put the condition back here after removing the `if` line. If that did happen, the develop-only dashboard data would be muddied with PRs and could need a manual cleaning or reset.